packages/twenty-front/src/pages/object-record/RecordIndexPageHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import { useParams } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+
+import { useObjectMetadataItemForSettings } from '@/object-metadata/hooks/useObjectMetadataItemForSettings';
+import { recordIndexViewTypeState } from '@/object-record/record-index/states/recordIndexViewTypeState';
+import { useIcons } from '@/ui/display/icon/hooks/useIcons';
+import { PageAddButton } from '@/ui/layout/page/PageAddButton';
+import { PageHeader } from '@/ui/layout/page/PageHeader';
+import { PageHotkeysEffect } from '@/ui/layout/page/PageHotkeysEffect';
+import { ViewType } from '@/views/types/ViewType';
+
+type RecordIndexPageHeaderProps = {
+  createRecord: () => void;
+};
+
+export const RecordIndexPageHeader = ({
+  createRecord,
+}: RecordIndexPageHeaderProps) => {
+  const objectNamePlural = useParams().objectNamePlural ?? '';
+
+  const { findObjectMetadataItemByNamePlural } =
+    useObjectMetadataItemForSettings();
+
+  const { getIcon } = useIcons();
+  const Icon = getIcon(
+    findObjectMetadataItemByNamePlural(objectNamePlural)?.icon,
+  );
+
+  const recordIndexViewType = useRecoilValue(recordIndexViewTypeState);
+
+  return (
+    <PageHeader
+      title={
+        objectNamePlural.charAt(0).toUpperCase() + objectNamePlural.slice(1)
```

Comment:
@charlesBochet there is a capitalize util function in packages/twenty-front/src/utils/string/capitalize.ts

Reply:
I've split a file in two so I missed it, but I'll use capitalize, thank you for pointing it